.github/CODEOWNERS

```diff
@@ -15,6 +18,7 @@ src/* @kheina @jaredcat @cooljeanius @rougetimelord
 .prettierrc @kheina @jaredcat
 CHANGELOG.md @kheina @rougetimelord
 LICENSE @kheina
+privacy.md @kheina @rougetimelord
```

Comment:
oh actually I guess you can feel free to add me to this one, too, since I copyedited it...

Reply:
Oops yea, I got you